### ISSUE TITLE
update DECIMER to CDK 2.9 (or better)

### ISSUE THREAD

**Issue by egonw** [cdk/nwo-openscience-2024#16]
* https://decimer.ai/

cc @Kohulan

**Comment by RvdPloeg** [comment]
The randepict code of decimer was upgraded to use cdk-2.9, see [link](https://github.com/RvdPloeg/RanDepict/compare/main...Steinbeck-Lab%3ARanDepict%3Amain).

**Comment by egonw** [comment]
@Kohulan, would it be that simple?

**Comment by Kohulan** [comment]
@egonw Yes. We are using the Jar directly. If there are no major changes to CDK then we don't have to change anything in the code.

**Comment by egonw** [comment]
@Kohulan, there should not be (m)any in 2.8 -> 2.9, but that is something we want to check and record. How can we best check this?

**Comment by Kohulan** [comment]
@egonw All tests have passed since our original implementation used CDK 2.8. The CDK functionalities we are currently using can be found at https://github.com/Steinbeck-Lab/RanDepict/blob/main/RanDepict/cdk_functionalities.py.

**Comment by egonw** [comment]
> The CDK functionalities we are currently using can be found at https://github.com/Steinbeck-Lab/RanDepict/blob/main/RanDepict/cdk_functionalities.py.

Okay, scanned it and it looks like there have not been APi in the used code here.

@RvdPloeg, a simple test would be to just run the methods. If there are any major API change, that will cause an immediate error.

**Comment by egonw** [comment]
The RanDepict update has been made, but the release with CDK 2.9/2.10 is (still) pending: https://github.com/OBrink/RanDepict/pull/57
